tests/system-tests/rdscore/internal/rdscorecommon/metallb-traffic-segregation.go

```diff
@@ -0,0 +1,731 @@
+			RDSCoreConfig.MetalLBFRRContainerNameTwo,
+			appURL)
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("second mockup app %s is not reachable from the container %s: %v",
+				appURL, RDSCoreConfig.MetalLBFRRContainerNameTwo, err))
+
+		By("Check the traffic flows for the second FRR through the expected interface")
+
+		for _, _pod := range stPodsList {
+			searchString := "10.18.124.21"
+
+			glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Making sure that the traffic flows for the second FRR "+
+				"through the %s interface", RDSCoreConfig.MetalLBTrafficSegregationTCPDumpIntTwo)
+
+			numberFound, logs, err := supporttools.ScanTCPDumpPodLogs(
+				APIClient,
+				_pod,
+				stNamespace,
+				stDeploymentLabel,
+				searchString,
+				timeStart)
+			Expect(err).ToNot(HaveOccurred(),
+				fmt.Sprintf("failed to tcpdump log results for the pod %s in namespace %s: %v",
+					_pod.Definition.Name, _pod.Definition.Namespace, err))
+
+			if numberFound == 0 {
+				glog.V(rdscoreparams.RDSCoreLogLevel).Infof("The searching string %s was not found in log: %s",
+					searchString, logs)
+			}
+
+			glog.V(rdscoreparams.RDSCoreLogLevel).Infof("The searching string %s was found in log: %s",
+				searchString, logs)
+		}
+	}
+}
+
+// VerifyMetallbMockupAppNotReachableFromOtherFRR test mockup app is not reachable from the not correct FRR container.
+//
+//nolint:funlen
+func VerifyMetallbMockupAppNotReachableFromOtherFRR(ctx SpecContext) {
+	By("Asserting if test URLs are provided")
+
+	if RDSCoreConfig.MetalLBFRROneIPv4 == "" && RDSCoreConfig.MetalLBFRROneIPv6 == "" {
+		glog.V(rdscoreparams.RDSCoreLogLevel).Infof(
+			"First MetalLB FRR IP not specified or are empty. Skipping...")
+		Skip("First MetalLB FRR IP not specified or are empty")
+	}
+
+	if RDSCoreConfig.MetalLBFRRTwoIPv4 == "" && RDSCoreConfig.MetalLBFRRTwoIPv6 == "" {
+		glog.V(rdscoreparams.RDSCoreLogLevel).Infof(
+			"Second MetalLB FRR IP not specified or are empty. Skipping...")
+		Skip("Second MetalLB FRR IP not specified or are empty")
+	}
+
+	if RDSCoreConfig.MetalLBTrafficSegregationTCPDumpIntOne == "" &&
+		RDSCoreConfig.MetalLBTrafficSegregationTCPDumpIntTwo == "" {
+		glog.V(rdscoreparams.RDSCoreLogLevel).Infof(
+			"One of the capture interfaces not specified or are empty. Skipping...")
+		Skip("One of the capture interfaces not specified or are empty")
+	}
+
+	By("Ensure LB application deployment created for the first bgp route")
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Searching for pods in %q namespace",
+		RDSCoreConfig.MetalLBLoadBalancerOneNamespace)
+
+	lbOnePodsList := findPodWithSelector(RDSCoreConfig.MetalLBLoadBalancerOneNamespace, lbDeploymentLabel)
+
+	if len(lbOnePodsList) == 0 {
+		glog.V(rdscoreparams.RDSCoreLogLevel).Infof(
+			"FRR One Load Balance app not found deployed. Skipping...")
+		Skip("FRR One Load Balance app not found deployed")
+	}
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Getting node name for the loadbalancer one application")
```

Comment:
where's the code to get the node name? assume smth like:
```
lbOneNodeName = lbOnePodsList[0].Object.Spec.NodeName
```

Reply:
it's not needed in this case, left-overs after debug; removed